domains/gabri.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "gablilli",
+    "email": "gabriele@lilli.eu.org"
+  },
+  "records": {
+    "NS": ["garrett.ns.cloudflare.com", "walk.ns.cloudflare.com"]
```

Comment:
Reason denied. Reason provided can be solved with Tailscale Funnel. Other reasons are invalid/not valid reasoning.